src/common/bitutil.h

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
To keep consistence with other files, could you rename this file to `bit_util.h`?

Reply:
done